Tools/CMake/AMReXThirdPartyLibraries.cmake

```diff
@@ -79,9 +79,10 @@ endif ()
 #
 if (AMReX_SUNDIALS)
     if (SUNDIALS_FOUND)
-        message(STATUS "SUNDIALS_FOUND is true, assuming nvecserial or gpu-specific vector found for version 5.7.0")
+        message(STATUS "SUNDIALS_FOUND is true, assuming nvecserial or gpu-specific vector found for version 6.0.0 or higher")
     else ()
-        find_package(SUNDIALS 5.7.0 REQUIRED)
+       set(SUNDIALS_MINIMUM_VERSION 6.0.0 CACHE INTERNAL "Minimum required SUNDIALS version")
```

Comment:
This is more of a curiosity. What is the reason to use a variable here?

Reply:
Previous code being reused? No strong reason